bundles/org.openhab.binding.mideaac/README.md

```diff
@@ -0,0 +1,76 @@
+## Midea AC Binding
+
+This binding integrates Air Conditioners that use the Midea protocol. Midea is an OEM for many brands.
+
+An AC device is likely supported if it uses one of the following Android apps or it's iOS equivalent.
+ 
+| Application						 		 |				Comment									 |
+|:-------------------------------------------|:------------------------------------------------------|
+|Midea Air (com.midea.aircondition.obm)		 | Full Support of key and token updates				 |
+|NetHome Plus (com.midea.aircondition)		 | Full Support of key and token updates				 |
+|SmartHome/MSmartHome (com.midea.ai.overseas)| Full Support of key and token updates				 |
+
+Note: The Air Conditioner must already be set-up on the WiFi network and must have a fixed IP Address 
+	with one of the three apps listed above for full discovery and key and token updates.
+
+## Supported Things
+
+This binding supports one Thing type `ac`.
+
+## Discovery
+
+Once the Air Conditioner is on the network (WiFi active) it could be discovered automatically.
+An IP broadcast message is sent and every responding unit gets added to the Inbox.
+
+## Binding Configuration
+
+No binding configuration is required.
+
+## Thing Configuration
+
+| Parameter            		| Required ? 		| Comment
```

Comment:
The Parameter column should contain the `camelCase` parameter id, reflecting the thing-type.xml / structure. If possible provide a default and or of it is marked advanced

Reply:
Improved, stilll userfull to add a column with defaults.
Als ow onder what  `Required for V.3` means, it is required yes/no